team.html

```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
             The current lead developers are <strong>Axel Donath</strong> and <strong>Régis Terrier</strong>.
         </p>
 
+        <h2 id="module-maintainer">Module maintainers</h2>
+          <p>Amongst the development team, some experts are devoted to the maintenance of some modules.</p>
+          
+          <p>They are in charge of:
+          <ul>
+            <li>supporting developers on tasks associated to their module(s),</li>
+            <li>making a review of proposed contributions for their module(s),</li>
```

Comment:
Reviewing contributions impacting the module

Reply:
Or following the astropy team description:
Evaluating new pull requests for quality, API consistency and Gammapy coding standards